aspnet/includes/coreWebAPI.md

```diff
@@ -0,0 +1,14 @@
+Consider using [ASP.NET Core Web API](/aspnet/core/web-api). It has the follow advantages over ASP.NET 4 Web API:
+
+- ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web apps on Windows, macOS, or Linux.
+- The ASP.NET Core MVC controller and Web API controller are unified.
+- Architected for testability.
+- Ability to develop and run on Windows, macOS, and Linux.
+- Open-source and community-focused.
+- Integration of modern, client-side frameworks and development workflows.
+- A cloud-ready, environment-based configuration system.
+- Built-in dependency injection.
+- A lightweight, high-performance, and modular HTTP request pipeline.
+- Ability to host on IIS, Nginx, Apache, Docker, or self-host in your own process.
```

Comment:
Shouldn't Kestrel be listed here too now that it's suitable to be used as a public-facing edge server?

Reply:
See https://github.com/aspnet/AspNetCore.Docs/pull/14204